inc/em_agent.h

```diff
@@ -92,6 +92,8 @@ class em_agent_t : public em_mgr_t {
      */
     bool send_action_frame(uint8_t dest_mac[ETH_ALEN], uint8_t *action_frame, size_t action_frame_len, unsigned int frequency=0, unsigned int wait_time_ms=0) override;
 
+    bool can_onboard_additions_aps();
```

Comment:
Can you add a small docstring + typo
```suggestion
    bool can_onboard_additional_aps();
```